os/using-systemd-and-udev-rules.md

```diff
@@ -0,0 +1,74 @@
+# Using systemd and udev Rules 
+
+In our example we will use libvirt VM. First of all we have to create systemd unit file `/etc/systemd/system/test.service`:
```

Comment:
Libvirt is one of our less popular platforms. Is there a way we can make this guide generic or use a different platform as the example?